data/categories.txt

```diff
@@ -2189,7 +2189,7 @@ ru:^Место для пикника|3Пикник
 ar:^مكان نزهة|سياحة|مناظر
 cs:3^Piknik|picnic|picnik
 da:3^Picnic
-nl:3^Picnicplaats|3picnicweide|3picnictafel
+nl:3^Picknickplaats|3picknickweide|3picknicktafel
```

Comment:
Are you sure all three synonyms need the "3" prefix? Do we need to suggest all three of them after typing 3 letters?

Reply:
You are right, only the first should be suggested. Fixed!